pillarbox-demo/src/main/java/ch/srgssr/pillarbox/demo/ui/showcases/integrations/cast/CastShowcase.kt

```diff
@@ -17,15 +16,12 @@ import androidx.compose.runtime.getValue
 import androidx.compose.ui.Alignment
 import androidx.compose.ui.Modifier
 import androidx.lifecycle.viewmodel.compose.viewModel
-import androidx.media3.common.util.RepeatModeUtil
-import androidx.media3.ui.PlayerView
 import ch.srgssr.media.maestro.MediaRouteButton
-import ch.srgssr.pillarbox.cast.PillarboxCastPlayer
 import ch.srgssr.pillarbox.demo.shared.data.samples.SamplesGoogle
 import ch.srgssr.pillarbox.demo.shared.data.samples.SamplesSRG
 import ch.srgssr.pillarbox.demo.shared.data.samples.SamplesUnifiedStreaming
+import ch.srgssr.pillarbox.demo.ui.player.DemoPlayerView
 import ch.srgssr.pillarbox.demo.ui.player.playlist.PlaylistView
-import ch.srgssr.pillarbox.ui.exoplayer.ExoPlayerView
 import androidx.compose.ui.graphics.Color as ComposeColor
```

Comment:
Now that we don't use `android.graphics.Color` anymore, you can remove the alias here.